aws/eks-customer/eks.tf

```diff
@@ -0,0 +1,101 @@
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "20.14.0"
```

Comment:
lets make version variable

Reply:
I believe this isn't possible due to terraform limitations. https://github.com/hashicorp/terraform/issues/28912